stacks/spica/src/passport/pages/apikey-index/apikey-index.component.ts

```diff
@@ -0,0 +1,38 @@
+import {Component, OnInit, TemplateRef, ViewChild} from "@angular/core";
+import {MatPaginator} from "@angular/material";
+import {ApiKey} from "src/passport/interfaces/apikey";
+import {Observable, of, Subject, merge} from "rxjs";
+import {switchMap, map} from "rxjs/operators";
+import {apiKeyService} from "src/passport/services/apikey.service";
+
+@Component({
+  selector: "app-apikey-index",
```

Comment:
Can you please change the selector to `passport-apikey-index` to preserve the convention.